feat: Mention collected data per integration (#69)
codacy/pulse-user-docs: docs/one-click-integrations/pagerduty-integration.md

1.  Follow the instructions on the Pulse UI to create a new webhook on PagerDuty using the provided URL. For example:

    ![PagerDuty webhook](images/pagerduty-webhook.png)

## Collected data

> After being configured, the PagerDuty integration will push data an it will be converted to [Pulse's data model](https://ingestion.pulse.codacy.com/v1/api-docs#tocs_event) to calculate the metrics shown on the UI. In the following table you can see how data from PagerDuty is used to calculate each of the metrics:
> 
> | Pulse Event | Source  | Fields                                              | Used in                                          |
> | ----------- | ------- | --------------------------------------------------- | ------------------------------------------------ |
> | Incident    | Commits | identifier → ???                                    | [Accelerate dashboard](../metrics/accelerate.md) |
> |             |         | timestampCreated → ???                              |                                                  |
> |             |         | timestampResolved → ???                             |                                                  |
> |             |         | system → *defined during the configuration process* |                                                  |

Reviewer: ```suggestion
After being configured, the integration, PagerDuty will push data and it will be converted into [Pulse's data model](https://ingestion.pulse.codacy.com/v1/api-docs#tocs_event) to calculate the metrics shown on the UI. In the following table you can see how data from PagerDuty is used to calculate each of the metrics:

| Pulse Event | Source     | Fields                                              | Used in                                          |
| ---------------- | ------------- | --------------------------------------------------- | ------------------------------------------------ |
| Incident       | pagerduty | identifier → id                                    | [Accelerate dashboard](../metrics/accelerate.md) |
|                    |                  | timestampCreated → created_at                              |                                                  |
|                    |                  | timestampResolved → last_status_change_at                             |                                                  |
|                                       |         | system → *defined during the configuration process* |                                                  |
```

1.  Follow the instructions on the Pulse UI to create a new webhook on PagerDuty using the provided URL. For example:

    ![PagerDuty webhook](images/pagerduty-webhook.png)

## Collected data

The table below lists the data that the PagerDuty integration collects from your PagerDuty webhooks, together with:

-   The mapping between the data collected from PagerDuty and the [Pulse data model](https://ingestion.pulse.codacy.com/v1/api-docs#tocs_event)
-   The metrics that Pulse calculates from the data to display on the dashboards

<table>
<thead>
<tr>
<th><strong>Data collected from PagerDuty</strong></th>
<th><strong>Mapping to Pulse data model</strong></th>
<th><strong>Used in</strong></th>
</tr>
</thead>
<tbody>
<tr>

>     <td>Incident</td>

Author: @rtfpessoa is this what it is called in PagerDuty?

Reviewer 2: Yes, this is the correct concept name on PagerDuty:

https://response.pagerduty.com/getting_started/#define-what-an-incident-and-major-incident-are-for-you

https://response.pagerduty.com/before/what_is_an_incident/

    <td>
        <p>Incident:</p>
        <ul>

>             <li><code>identifier</code>: id</li>
>             <li><code>timestamp_created</code>: created_at</li>
>             <li><code>timestamp_resolved</code>: last_status_change_at</li>
>             <li><code>system</code>: <i>defined during the configuration process</i></li>

Reviewer 2: I couldn't find the documentation for the PagerDuty webhooks, but would it make sense to include more "human-readable" descriptions for the data that we obtain from PagerDuty instead of what I'm assuming are the webhook field names?

```suggestion
            <li><code>identifier</code>: incident ID</li>
            <li><code>timestamp_created</code>: incident creation date</li>
            <li><code>timestamp_resolved</code>: incident last status change date</li>
            <li><code>system</code>: <i>defined during the configuration process</i></li>
```

Reviewer: @prcr those fields are the webhook fields

Reviewer 2: Taking @rtfpessoa's [suggestion below](https://github.com/codacy/pulse-user-docs/pull/69#issuecomment-896690391) into account:

```suggestion
            <li><code>identifier</code>: <code>id</code> (incident ID)</li>
            <li><code>timestamp_created</code>: <code>created_at</code> (date when the incident was first triggered)</li>
            <li><code>timestamp_resolved</code>: <code>last_status_change_at</code> (date of the last status change of the incident)</li>
            <li><code>system</code>: <i>defined during the configuration process</i></li>
```
